spec/requests/api/person_escort_records_controller_update_spec.rb

```diff
@@ -6,6 +6,12 @@
   describe 'PATCH /person_escort_records/:person_escort_record_id' do
     include_context 'with supplier with spoofed access token'
 
+    subject(:patch_person_escort_record) do
```

Comment:
This is a new style for making the request. Currently we have:

1. Directly in before block
2. Directly in before block with skip before
3. Not in before block (and just in examples)
4. Directly in the before block with method
5. Directly in the before block with method and skip before
6. Directly in the before block with subject let

I'm not saying this is a bad way to do it. Just that we have so many permutations of making a request in our request specs 🤔

Reply:
I don't think it's a new style?  see e.g. `/api/v1/person_escort_records/#{person_escort_record_id}"` ?

Reply:
Ah, in PRs I've reviewed.